### ISSUE TITLE
Log out button right after signing up

### ISSUE THREAD

**Issue by imorente** [netlify/netlify-identity-widget#85]
It was unexpected to end up on this screen right after signing up:
<img width="426" alt="screen shot 2017-10-20 at 3 16 01 pm" src="https://user-images.githubusercontent.com/83225/31843877-43c75cc2-b5aa-11e7-94da-442058f087eb.png">

My expectation was:
1. Follow invite link from email
2. Set password (or register with an external provider)
3. Get redirected to admin dashboard

**Comment by bcomnes** [comment]
Agreed, this is a weird flow.  https://github.com/netlify/netlify-identity-widget/issues/81 may possibly improve.

**Comment by DavidSabine** [comment]
I suggest re-opening this.  #81 appears to have been abandoned and is, actually, an unrelated concern.

This issue is specifically about the user's experience after they log in.  Presenting the user with a giant call to action button on the modal which reverses the very action they just finished (logging in) is #badUX.  This problem makes this otherwise elegant solution **unusable**.

**Comment by sw-yx** [comment]
@DavidSabine not to take away from your critique but as i'm aware you're working on a Gatsby + Netlify app right now may i suggest [one of the alternative bindings to Netlify Identity/GoTrue](https://github.com/netlify/netlify-identity-widget#list-of-alternatives) so that you have more control over your UX. `netlify-identity-widget` will forever be an opinionated drop in widget, if you want more control its probably a good idea to drop down one level.

**Comment by DavidSabine** [comment]
Thank you @sw-yx, I may look into GoTrue.  Thanks for that.

Regardless, I maintain that the original issue opened by @imorente hasn't really been addressed.  The model would be much improved with a minor change in the UI.

**Comment by laurajodz** [comment]
FYI Devex owns this repo but doesn't use GitHub project boards like Eng does. So there is no project to add to.

(https://netlify.slack.com/archives/CCC1HDWQY/p1600806294044200)

**Comment by fabled228** [comment]
i had run into the same problem. I am using netlify-identity-widget and when user successfully logs in by google account after returning to the app, login widget 'logged in as' appears for 1-2sec and then hides, but leaves empty <body></body>, so user can't press anything and this disappears after page reloading. Can anyone help my with this problem?
